<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowBulletViewer</class>
 <widget class="QMainWindow" name="MainWindowBulletViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simox SimDynamics Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Robot</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>381</width>
         <height>261</height>
        </rect>
       </property>
       <property name="title">
        <string>Joint Info</string>
       </property>
       <widget class="QLabel" name="label_RNTarget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Joint target: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_RNVelocity">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>221</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Joint velocity: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_RNValue">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>221</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Joint value: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_RNName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Name: &lt;not set&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_RNPosVisu">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>261</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>VISU (simox):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ForceTorqueA">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>351</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ForceTorqueA: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_RNPosGP">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>261</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>GlobalPose (simox):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_RNVelocityTarget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>241</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Joint velocity target: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ForceTorqueJoint">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>351</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ForceTorqueJoint: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_RNPosCom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>261</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Com</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ForceTorqueB">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>361</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ForceTorqueB: </string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>610</y>
         <width>381</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Visualization</string>
       </property>
       <widget class="QCheckBox" name="checkBoxCom">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>CoM</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxContacts">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Contacts</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxColModel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Collision Model</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxAntiAliasing">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>40</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>20</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Anti Aliasing</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>371</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Physics</string>
       </property>
       <widget class="QPushButton" name="pushButtonStartStop">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>20</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop Engine</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonStep">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>20</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Step Engine</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderUpdateTimer">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_TargetMax_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>110</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>200ms</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_TargetMin_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>110</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>5ms</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>70</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Physics Update timer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_TargetMax_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>33ms</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fixed Time Step</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderFixedTimeStep">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>33</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_TargetMin_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>1ms</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxFixedTimeStep">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Fixed Time Step</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>361</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Joints</string>
       </property>
       <widget class="QCheckBox" name="checkBoxActuation">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Joint actuation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxRobotNode">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RobotNode</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_TargetMin">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>-1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_TargetMax">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>90</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderTarget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>241</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Target Joint Value</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonAddObject">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Object</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
